modules/storage/cosmos-db/modules/sql_containers.bicep

```
@@ -0,0 +1,74 @@
+param cosmosDBAccountName string
+param enableServerless bool = false
+param sqlDatabaseName string
+param sqlDatabaseContainer object
+
+resource cosmosDBAccount 'Microsoft.DocumentDB/databaseAccounts@2022-11-15' existing = {
+  name: cosmosDBAccountName
+}
+
+resource cosmosDBAccountSqlDatabase 'Microsoft.DocumentDB/databaseAccounts/sqlDatabases@2022-11-15' existing = {
+  name: sqlDatabaseName
+  parent: cosmosDBAccount
+}
+
+var varStoredProcedures = contains(sqlDatabaseContainer, 'storedProcedures') ? sqlDatabaseContainer.storedProcedures : []
+var varTriggers = contains(sqlDatabaseContainer, 'triggers') ? sqlDatabaseContainer.triggers : []
+var varUserDefinedFunctions = contains(sqlDatabaseContainer, 'userDefinedFunctions') ? sqlDatabaseContainer.userDefinedFunctions : []
+
+resource cosmosDBAccount_sqlDatabasesqlDatabaseContainers 'Microsoft.DocumentDB/databaseAccounts/sqlDatabases/Containers@2022-11-15' = {
```

```suggestion
resource sqlDatabaseContainers 'Microsoft.DocumentDB/databaseAccounts/sqlDatabases/Containers@2022-11-15' = {
```

I dont think you need the long resource names, and could trim the prefixs. Particularly consider triming the double 'sqlDatabasesqlDatabase'

---

removed `cosmosDBAccount` prefix from all resource definitions under modules. Also trimmed the param names at other places.